src/test/java/com/unstoppabledomains/ProxyReaderTest.java

```diff
@@ -0,0 +1,40 @@
+package com.unstoppabledomains;
+
+import com.unstoppabledomains.resolution.contracts.cns.ProxyReader;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigInteger;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ProxyReaderTest {
+
+    private static final String URL = "https://mainnet.infura.io/v3/781c1e5cae32417b93eac26042950d25";
```

Comment:
seems like we commit a token of `mainnet.infura.io` Is this OK for open source solution?
may be better to use link similar as we use in resolution swift

`https://main-rpc.linkpool.io`

Reply:
Good point, thanks. This will be addressed in the scope of another story.